go/status/status.go

```diff
@@ -0,0 +1,244 @@
+package status
+
+import (
+	"errors"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/keybase/client/go/install"
+	"github.com/keybase/client/go/libkb"
+	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+)
+
+func GetCurrentStatus(mctx libkb.MetaContext) (res keybase1.CurrentStatus, err error) {
+	cr := mctx.G().Env.GetConfig()
+	if cr == nil {
+		return res, nil
+	}
+	res.Configured = true
+	if uid := cr.GetUID(); uid.Exists() {
+		res.Registered = true
+		res.User = libkb.NewUserThin(cr.GetUsername().String(), uid).Export()
+	}
+	res.SessionIsValid = mctx.ActiveDevice().Valid()
+	res.LoggedIn = res.SessionIsValid
+	return res, nil
+}
+
+func GetClientStatus(mctx libkb.MetaContext) (res []keybase1.ClientStatus) {
```

Comment:
is this different from `libkb.GetClientStatus`?

Reply:
oops no this is supposed to be removed since `install` needs this function